src/main/java/org/traccar/model/Position.java

```diff
@@ -44,6 +44,9 @@ public class Position extends Message {
     public static final String KEY_ODOMETER_TRIP = "tripOdometer"; // meters
     public static final String KEY_HOURS = "hours"; // milliseconds
     public static final String KEY_STEPS = "steps";
+    public static final String KEY_BLOOD_OXYGEN = "bloodOxygen";
+    public static final String KEY_PRESSURE_HIGH = "pressureHigh";
+    public static final String KEY_PRESSURE_LOW = "pressureLow";
```

Comment:
This is a comment from previous PR:

> We only add new parameters here where they are used by at least 2 protocols.

Reply:
Okay. I will set those parameters as custom parameters. So i ll hardcode them as they was in the first commit.
And I ll close the pull request in the web app.
is this ll be ok?

Reply:
That should be ok and then please close the other PR.